gammapy/modeling/models/spectral.py

```diff
@@ -316,11 +319,29 @@ def plot(
 
         energy_min, energy_max = energy_range
         energy = MapAxis.from_energy_bounds(
-            energy_min, energy_max, n_points, energy_unit
-        ).edges
+                                            energy_min, energy_max, n_points, energy_unit
+                                            ).edges
+
+        if sed_type:
```

Comment:
I think the `if sed_type` block has no effect, just remove...

Reply:
OK.